lib/tasks/index.rake

```diff
@@ -5,6 +5,106 @@ require "octokit"
 require "time"
```

Comment:
I see an obvious set of changes here. Please make sure the setup guide still works or update it at https://github.com/git/git-scm.com#setup

(I'm also going to run it locally to help checking any missing steps)